src/features/bookOffer/services/utils.ts

```diff
@@ -0,0 +1,50 @@
+import { OfferStockResponse } from 'api/gen'
+import { formatToSlashedFrenchDate } from 'libs/dates'
+import { CENTS_IN_EURO } from 'libs/parsers/pricesConversion'
+
+const EURO_SYMBOL = '€'
+
+export const getStocksByDate = (
+  stocks: OfferStockResponse[]
+): { [date: string]: OfferStockResponse[] } => {
+  const stockDates: { [date: string]: OfferStockResponse[] } = {}
+
+  stocks.forEach((stock) => {
+    if (stock.beginningDatetime !== null && stock.beginningDatetime !== undefined) {
+      const formattedDate = formatToSlashedFrenchDate(stock.beginningDatetime.toString())
+      stockDates[formattedDate] = stockDates[formattedDate]
+        ? stockDates[formattedDate].concat(stock)
+        : [stock]
+    }
+  })
+
+  return stockDates
+}
+
+export enum OfferStatus {
+  BOOKABLE = 'BOOKABLE',
+  NOT_BOOKABLE = 'NOTBOOKABLE',
```

Comment:
```suggestion
  NOT_BOOKABLE = 'NOT_BOOKABLE',
```

c'était pour les deux, au temps pour moi :)

Reply:
je fais la modif